scheduler/project.clj

```diff
@@ -162,7 +162,10 @@
     :source-paths []}
 
    :test-console
-   {:jvm-opts ["-Dcook.test.logging.console"]}}
+   {:jvm-opts ["-Dcook.test.logging.console"]}
+   :docker
+   ; avoid calling javac in docker
```

Comment:
Can we add onto this comment that the reason this is OK is that the Java file is only used by unit tests?